src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software': '8.13.8',
+	'confluence': '7.12.2',
+	'stash': '7.12.1',
+}
+
+
+def get_lts_version(argv):
+	product = argv[0].lower()
+	if product == 'jira':
+		product = 'jira-software'
+	elif product == 'bitbucket':
+		product = 'stash'
+
+	if product in known_lts_version:
+		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+
+		try:
+			fl = urllib.request.urlopen(url)
+			fdata = fl.read()
+			jsdata = json.loads(fdata[10:len(fdata)-1].decode("utf-8"))
+			enterprise_edition = [x for x in jsdata if x['edition'].lower() == 'enterprise']
+			sortedData = sorted(enterprise_edition, key=lambda k:cversion(k['version']), reverse=True)
+
+			if len(sortedData) > 0:
+				lts_version = sortedData[0]['version']
+			else:
+				lts_version = known_lts_version[product]
+
+			# as currently latest lts versoin of bitbucket and confluence don't support k8s
+			# we use none-lts version of those products in the test
+			if cversion(lts_version) < cversion(known_lts_version[product]):
+				lts_version = known_lts_version[product]
+		except:
+			lts_version = known_lts_version[product]
+
+		lts_version = f"{lts_version}-jdk11"
+	else:
+		lts_version = 'unknown'
+
+	return lts_version
+
+
+def cversion(version):
+	vers = [int(v) for v in version.split(".")]
+	mapped_ver = ''
+	for i in range(max(len(vers)-1, 4)):
+		if len(vers) > i:
+			mapped_ver = f'{mapped_ver}{str(vers[i]).zfill(5)}'
+		else:
+			mapped_ver = f'{mapped_ver}00000'
```

Comment:
Same as comment above, why we set `mapped_ver` to `f'{mapped_ver}00000'` isnt overly clear. It might also be better in a named function.

Reply:
This is for the case that the version missing minor, patch, or build - so replace with '00000' to makes the version unified for string comparison:

'7.12'             => '0000700120000000000'
'7.12.39'        => '0000700120003900000' 
'7.12.39.124' => '0000700120003900124'

Reply:
Neat. But I think we should use a librabry for this, its much clearer:

```
>>> from packaging import version
>>> version.parse("2.3.1") < version.parse("10.1.2")
True
```
> https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Reply:
I have implemented by this approach initially, but this is not the best for our case so I reverted this approach and refactored this part. This brings a dependency to packaging module which is not installed by default in ubuntu build server. If we go with this approach we need to install additional one which I preferred to add cversion method to script rather than add additional dependency.